civiform/civiform
# [Program Card Images] Add CTA from program image upload page to translations page

### Description

For program images, admins need to provide an alt-text description of the image and that alt-text needs to be translated to all the languages supported by the CE. This PR adds a button from the program image page to the translations page to encourage admins to add translations.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. Enable `PROGRAM_CARD_IMAGES` flag.
2. As an admin, navigate to the program images upload page.
3. Verify there's a "Manage translations" button that's disabled if there's no description set but enabled if there is a description.
4. Click the "Manage translations" button and verify it takes you to the translations page. Verify that the translations page has a entry for the program description to be translated into each locale.

### Issue(s) this completes

Fixes #6001 

Part of epic #5676
